crates/ruma-events/src/enums.rs

```diff
@@ -6,120 +6,117 @@ use crate::{from_raw_json_value, EventDeHelper};
 
 event_enum! {
     /// Any global account data event.
-    kind: GlobalAccountData,
-    events: [
-        "m.direct",
-        "m.ignored_user_list",
-        "m.push_rules",
-    ]
-}
+    enum GlobalAccountData {
+        events: [
+            "m.direct",
+            "m.ignored_user_list",
+            "m.push_rules",
+        ]
+    }
```

Comment:
Can we remove the extra `events: []` layer?

```suggestion
    enum GlobalAccountData {
        "m.direct",
        "m.ignored_user_list",
        "m.push_rules",
    }
```